doc/explanation/networks.md

```diff
@@ -28,11 +28,11 @@ You can configure the network device in one of the following ways:
 
         lxc config device add <instance_name> eth0 nic nictype=bridged parent=br0
 - {doc}`Create a managed network </howto/network_create>` and add it as a network device to your instance.
-  With this method, LXD has all required information about the configured network, and you only need to specify the network name when adding the device:
+  With this method, LXD has all required information about the configured network, and you can directly attach it to your instance as a device:
 
-        lxc config device add <instance_name> <device_name> nic network=<network_name>
+        lxc network attach <device_name> <instance_name>
```

Comment:
This should be `lxc network attach <network_name> <instance_name> <device_name>`.

Reply:
Although `<device_name>` is optional I think its worth recommending using that option so we encourage users to add the interface named as `eth0` rather than adding a NIC that will be named after the network, and so by default causing it to appear as an interface called `<network_name>` in their container.

Reply:
Oh of course, `network_name` instead of `device_name`. But `device_name` is optional, right?